gamenews

```diff
@@ -1,4 +1,4 @@
-2025-02-20 12-30-00:
-	Title: Playtest 20250220
+2025-03-03 16-00-00:
+	Title: Release 20250303
 	Author: PunkPun
-	Content: We present release candidate Playtest 20250220\n\nThanks everyone for taking part in the playtests! The candidate includes but is not limited to: improved map editor, higher quality Dune 2000 assets, significant performance improvements, new missions, an ingame EVA database.\n\nIf you find any critical issues don't forget to share your feedback with us on our forum, community Discord server, or GitHub. Good luck on the battlefield, Commanders!
+	Content: A new stable OpenRA Release 20250303 is now available! Featuring newly revamped Map Editor, a switch to high-quality Dune 2000 assets, significant performance improvements, new missions, an ingame EVA database.
```

Comment:
This ends rather abruptly. How about an "See the full news post at www.openra.net for more information and the release download!" or similar?